### PR TITLE
Add heart to CocktailDetail

### PR DESCRIPTION
**Pull request by HeloiseFerla**

### CODE REVIEW

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: src/components/CocktailCard/style.css [hunk 1]
```diff
@@ -87,6 +87,8 @@
 }
 @media only screen and (min-width: 1600px) {
   .mainCard {
-    width: 500px;
+    min-width: 400px;
+    max-width: 500px;
+    /* ou width : 500px;*/
```
Comment (Hacene010) [inline src/components/CocktailCard/style.css]: Dead code.

**Review by Hacene010 — CHANGES_REQUESTED** [review]
I'm not sure your SM reviewed this PR ???

**Comment by LoicBrassart** [comment]
No SM validation

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**